docs/user/rest-api.rst

```diff
@@ -690,6 +698,13 @@ List Locations
 
     GET /api/v1/controller/location/
 
+.. _location_list_estimated:
+
+**Estimated Status**
+
+If :doc:`Estimate Location feature <estimated-location>` is enabled, each
+location in the response will also include a ``is_estimated`` field.
+
```

Comment:
These details are used multiple times in this page. Can we declare them once and import?